src/components/sidebar/Sidebar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './Sidebar.css'
+
+function Sidebar() {
+  return (
+    <>
+      <nav className="sidebar">
+        <div className="nav-links">
+          <a className="sidebar-link">Link1</a>
```

Comment:
lets not put static links instead go for config based renderer.
We can discuss in detail about this.

Reply:
Okay, let's discuss.